defaults/main.yml

```
@@ -52,7 +52,7 @@ infiniband_mlnx_ofed_flavor: 'basic'
 infiniband_configure_repos: True
 infiniband_install_kernel_modules: True
 infiniband_install_mlnx_ofed: True
-
+infiniband_update_mlnx_ofed: False
```

This name could be misleading. I would interpret it as "Update mlnx_ofed packages".

The purpose of your change is to build the mlnx-ofed-kernel modules with dkms during kernel headers post-install before any other module.

An example that may seem more self-explanatory.
```suggestion
infiniband_dkms_postinst: False
```

I guess that would need some comments in the defaults/main.yml file and the README to explain the purpose of this variable.